test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Parsers/UriQueryExpressionParserTests.cs

```diff
@@ -653,6 +653,30 @@ public void ParseApplyWithNestedFunctionAggregation()
             funcToken.Name.ShouldBeEquivalentTo("cast");
         }
 
+        [Fact]
+        public void ParseApplyWithComputeWithMathematicalOperations()
+        {
+            string compute = "compute(Prop1 mul Prop2 as Product,Prop1 div Prop2 as Ratio,Prop2 mod Prop2 as Remainder)";
+            ComputeToken token = this.testSubject.ParseApply(compute).First() as ComputeToken;
+            token.Kind.ShouldBeEquivalentTo(QueryTokenKind.Compute);
+            List<ComputeExpressionToken> tokens = token.Expressions.ToList();
+            tokens.Count.Should().Be(3);
+            tokens[0].Kind.ShouldBeEquivalentTo(QueryTokenKind.ComputeExpression);

```

Comment:
Maybe it's more clear to create a private method to verify # 1, # 2, # 3 separately.

Reply:
We have test ParseComputeWithMathematicalOperations that checks $compute for that case. I'm adding the same for compute()